File: FIL/FIL.Web.Core/ViewModels/Login/LoginWithOTPResponseModel.cs

namespace FIL.Web.Core.ViewModels.Login
{
    public class LoginWithOTPResponseModel
    {
        public bool Success { get; set; }
        public bool? IsAdditionalFieldsReqd { get; set; } //for new user  
        public bool? EmailAlreadyRegistered { get; set; }
        public Contracts.Models.User User { get; set; }
        public SessionViewModel Session { get; set; }
    }
}

File: shared/models/LoginWithOTPResponseModel.ts
import { SessionViewModel } from "./SessionViewModel";
import { UserViewModel } from "./UserViewModel"


export class LoginWithOTPResponseModel {
    phoneCode: string;
    phoneNumber: string;
    isAdditionalFieldsReqd?: boolean;
    emailAlreadyRegistered?: boolean;
    success: boolean;
    session: SessionViewModel;
    user: UserViewModel;
}
